app/markets/ActivityTable.tsx

```diff
@@ -80,35 +73,42 @@ export const ActivityTable = ({ id }: { id: string }) => {
           <LoadingSkeleton />
         ) : (
           <TableBody className="text-base font-semibold">
-            {activities.map(activity => (
-              <TableRow key={activity.transactionHash}>
-                <TableCell className="pl-4 text-text-high-em">
-                  <a
-                    href={getGnosisAddressExplorerLink(activity.user.id)}
-                    className="hover:underline"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    {shortenAddress(activity.user.id)}
-                  </a>
-                </TableCell>
-                <TableCell>
-                  <Tag
-                    size="xs"
-                    colorScheme={getTagColorScheme(activity.transactionType)}
-                    className="w-fit uppercase"
-                  >
-                    {activity.transactionType}
-                  </Tag>
-                </TableCell>
-                <TableCell className="truncate text-text-high-em">
-                  {formatEtherWithFixedDecimals(activity.sharesOrPoolTokenAmount)}
-                </TableCell>
-                <TableCell className="text-nowrap pr-4 text-right text-text-low-em">
-                  {formatDateTime(activity.creationTimestamp)}
-                </TableCell>
-              </TableRow>
-            ))}
+            {activities?.map(activity => {
+              const outcomes = activity.fpmm.outcomes;
+              const outcomeIndex: 0 | 1 = activity.outcomeIndex;
```

Comment:
Let's use `Outcome` class here. And let's use its attributes below, like `symbol`.

We just need to instantiate the Outcome class equal to `outcomeIndex`.